guidelines.md

```diff
@@ -0,0 +1,28 @@
+# Guidelines
+## Writing
+
+When writing for ipfs, be it commit messages or posting issues, here's a collection of things to take note of.
+
+### Ungendering
+
+#### Motivation
+
+To avoid gendering in community interaction has multiple benefits. Avoiding the alienating feeling of documentation implicitly misgendering you, "the user and his computer". This is true of roughly half the population. Using plural forms is a good hack on the english language to get around this problem. This also includes extraterrestial beings and future lifeforms.
```

Comment:
I agree: guidelines should include some basic pronoun examples.
That way reader does not have to look it up elsewhere.